Makefile

```diff
@@ -15,10 +15,15 @@ lint: ## Runs linting on the go files. Requires docker to be installed
 	@docker run --rm -v $(CURDIR):/go-auth0 -w /go-auth0 golangci/golangci-lint:v1.44.0 golangci-lint -c .golangci.yaml run ./...
 
 .PHONY: test
-test: ## Runs tests
+test: ## Runs tests without vcr recordings
 	@echo "Running tests..."
 	@go test -cover -covermode=atomic -coverprofile=coverage.out ./...
 
+.PHONY: test-vcr
+test-vcr: ## Runs tests with vcr recordings
```

Comment:
Note: Although we are adding a `make test-vcr` command, we aren't using it yet within the CI. In this PR we're still going to run against the real E2E tests. 

Only after this and the following recordings PR is approved, we are going to modify the CI to use them. 

PR2: https://github.com/auth0/go-auth0/pull/68

Reply:
Sure, that makes sense.